locales/fr.yaml

```diff
@@ -303,11 +303,11 @@ created: Créé
 yes: Oui
 no: Non
 wizard_add_invoice_init: |
-  🤖 J'ai besoin que vous m'envoyiez une facture de ${satsAmount} satoshis qui vaut ${fiatAmount} ${devise} afin que je puisse vous envoyer les fonds.
+  🤖 J'ai besoin que vous m'envoyiez une facture de ${satsAmount} satoshis qui vaut ${fiatAmount} ${currency} afin que je puisse vous envoyer les fonds.
 
   Si vous n'envoyez pas la facture dans les ${expirationTime} minutes, la commande sera annulée.
 wizard_add_invoice_exit: |
-  Vous avez quitté le mode assistant, vous pouvez maintenant écrire les commandes et ajouter une facture à l'offre avec la commande /setinvoice indiquant l'ID de la commande et la facture. Vous pouvez m'envoyer une facture de ${montant} satoshis, mais j'accepte également les factures sans montant spécifié :
+  Vous avez quitté le mode assistant, vous pouvez maintenant écrire les commandes et ajouter une facture à l'offre avec la commande /setinvoice indiquant l'ID de la commande et la facture. Vous pouvez m'envoyer une facture de ${Amount} satoshis, mais j'accepte également les factures sans montant spécifié :
```

Comment:
please change `Amount` to `amount`, everything in code developing is case sensitive